includes/functions/functions_taxes.php

```diff
@@ -166,9 +166,9 @@ function zen_add_tax($price, $tax) {
 
     if (IS_ADMIN_FLAG === true) {
       if (DISPLAY_PRICE_WITH_TAX_ADMIN == 'true') {
-        return zen_round($price, $currencies->currencies[DEFAULT_CURRENCY]['decimal_places']) + zen_calculate_tax($price, $tax);
+        return $price + zen_calculate_tax($price, $tax);
       } else {
-        return zen_round($price, $currencies->currencies[DEFAULT_CURRENCY]['decimal_places']);
+        return $price; 

```

Comment:
I'm assuming these changes are here to mirror the `else` condition on the following lines.